src/terrain/util/config.clj

```diff
@@ -174,7 +174,7 @@
 (cc/defprop-optstr notificationagent-base-url
   "The base URL to use when connecting to the notification agent."
   [props config-valid configs notification-routes-enabled]
-  "terrain.notificationagent.base-url" "http://notification-agent")
+  "terrain.notificationagent.base-url" "http://notifications")
```

Comment:
This is the base URL used by the new notifications service in Kubernetes.

Reply:
Would it be worth changing the property and `defprop` name here as well, presumably to `terrain.notifications.base-url` and `notifications-base-url`? That would be more consistent with our other services, so it's one fewer "oh, right, it's named that because of <ancient history>" thing to deal with in the future.

If it's out of scope for this PR though, that's fine, just came to mind.

Reply:
That's a good idea. I'll update them as well.